README.md

```diff
@@ -61,7 +61,8 @@ image_name:
       volume: ... # docker volume if needed, format: localdir:/path/to/mount
       cmd: [...]  # shell commands run to be sure tools are well installed
     build_args: [...]  # If ARG are defined in Dockerfile
-    template_vars: [...]  # If templated Dockerfile
+    template_vars:  # If templated Dockerfile
+      VERSION: # If multiple image with different based image version is created
```

Comment:
This is not really mandatory I think : any image is very much free to name its own variables. Unless we decide to find a generic name for the version of the base image of course.

Reply:
IMHO `If the templated Dockerfile describes multiple images with different base images` is a bit better

Reply:
The comment about `VERSION` will be deleted, it was used when I wanted to build multiple image with different base version (python 3.6, 3.7 for the same tool version)